src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj

```diff
@@ -35,6 +35,7 @@
     </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
+    <Reference Include="PresentationFramework.Aero2" />
```

Comment:
based on what i read online, i think Aero2 is only shipped in win8 and up, while Aero is shipped win7 & up. So i would say, let's just change this to aero, rest everything looks good

Reply:
@rohit21agrawal I tried opening a new experimental instance and looking at the loaded modules and it seemed like only `Aero2` was loaded, therefore if we change this to `Aero` we would be loading an extra dll for Win8+. @rrelyea raised a concern about tying our implementation to a specific theme, will this make our design outdated if in the future windows changes the theme to something other than `Aero`?

 I also saw that it might be possible to change the template to use a normal border instead of the `ListViewChrome`, I will try to do that and see if it work.